mlrun/datastore/azure_blob.py

```diff
@@ -26,37 +27,58 @@
 # Azure blobs will be represented with the following URL: az://<container name>. The storage account is already
 # pointed to by the connection string, so the user is not expected to specify it in any way.
 
+MAX_CONCURRENCY = 100
+MAX_BLOCKSIZE = 1024 * 1024 * 4
+MAX_SINGLE_PUT_SIZE = 1024 * 1024 * 8  # for BlobServiceClient only.
```

Comment:
```suggestion
MAX_SINGLE_PUT_SIZE = 1024 * 1024 * 8  # for BlobServiceClient only
```
aside from the period at the end, this comment doesn't really clarify when this setting has an effect and when it does not.